ai_agents/devable_research_agent/scripts/run_pipeline.py

```
@@ -0,0 +1,488 @@
+
+
+
+def _run_step(name: str, cmd: list) -> bool:
+
+    _log(f"START {name}")
+
+    _log(f"CMD {' '.join(cmd)}")
+
+    try:
+
+        result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)